File: API/ProcessInstance/Objects/ProcessParam.cs
using API.ENUMS;

namespace API.ProcessInstance.Objects
{
    public class ProcessParam
    {
        public string Name { get; set; }
        public string Code { get; set; }
        public ProcessParamType ParamType { get; set; }
        public string Condition { get; set; }
        public string stringParam { get; set; }
        public int intParam { get; set; }
        public double doubleParam { get; set; }
        public bool boolParam { get; set; }
    }
}

File: Client/src/app/Interfaces/ProcessParam.ts
import { DTOStatus } from "../Enums/DTOStatus";

export interface ProcessParam
{
    id: number;
    name: string;
    code: string;
    paramRouteType: number;
    paramType: number;
    condition: string;
    stringParam: string;
    intParam: number;
    doubleParam: number;
    boolParam: boolean;
    status: DTOStatus;
}